src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/IPackageMetadata.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using NuGet.Versioning;
+
+namespace NuGet.Packaging
+{
+    public interface IPackageMetadata
```

Comment:
This interface is implemented by both PackageBuilder and ManifestMetadata, so a rename doesn't seem appropriate